src/registrar/templates/home.html

```diff
@@ -138,7 +138,7 @@ <h2>Domain requests</h2>
       aria-live="polite"
     ></div>
     {% else %}
-    <p>You don't have any active domain requests right now</p>
+    <p>You don't have any domain requests yet</p>
```

Comment:
what no sad face emoji?

Reply:
I was trying to match the wording in the Domains table, but I think your suggestion makes sense. What do you think of the word "registered" in the Domains message?

You don't have any **registered** domains yet

Reply:
I can't think of anything better, but we should explain "registered" in the help content. (I don't know what that means technically. I would remove "yet" in all cases.

Reply:
@michelle-rago Where does it say "Your domain applications?"

Reply:
Oh, just realized it is for the screenreader. `<caption class="sr-only">Your domain applications</caption>